src/components/KuadrantPoliciesPage.tsx

```diff
@@ -0,0 +1,422 @@
+  const { t } = useTranslation();
+
+  const columns: TableColumn<K8sResourceCommon>[] = [
+    {
+      title: t('plugin__console-plugin-template~Name'),
+      id: 'name',
+      sort: 'metadata.name',
+      transforms: [sortable],
+    },
+    {
+      title: t('plugin__console-plugin-template~Type'),
+      id: 'type',
+      sort: 'kind',
+      transforms: [sortable],
+    },
+    {
+      title: t('plugin__console-plugin-template~Namespace'),
+      id: 'namespace',
+      sort: 'metadata.namespace',
+      transforms: [sortable],
+    },
+    {
+      title: t('plugin__console-plugin-template~Status'),
+      id: 'Status',
+    },
+    {
+      title: t('plugin__console-plugin-template~Created'),
+      id: 'Created',
+      sort: 'metadata.creationTimestamp',
+      transforms: [sortable],
+    },
+    {
+      title: '',  // No title for the kebab menu column
+      id: 'dropdown-with-kebab',
+      props: { className: 'pf-v5-c-table__action' },
+    },
+  ];
+
+  const AllPolicyRow: React.FC<RowProps<K8sResourceCommon>> = ({ obj, activeColumnIDs }) => {
+    const [group, version] = obj.apiVersion.includes('/') ? obj.apiVersion.split('/') : ['', obj.apiVersion];
+    return (
+      <>
+        <TableData id={columns[0].id} activeColumnIDs={activeColumnIDs}>
+          <ResourceLink groupVersionKind={{group: group, version: version, kind: obj.kind}} name={obj.metadata.name} namespace={obj.metadata.namespace} />
+        </TableData>
+        <TableData id={columns[1].id} activeColumnIDs={activeColumnIDs}>{obj.kind}</TableData>
+        <TableData id={columns[2].id} activeColumnIDs={activeColumnIDs}>
+          <ResourceLink groupVersionKind={{version: "v1", kind: "Namespace"}} name={obj.metadata.namespace} />
+        </TableData>
+        <TableData id={columns[3].id} activeColumnIDs={activeColumnIDs}>{statusConditionsAsString(obj)}</TableData>
+        <TableData id={columns[4].id} activeColumnIDs={activeColumnIDs}>
+          <Timestamp timestamp={obj.metadata.creationTimestamp} />
+        </TableData>
+        <TableData id={columns[5].id} activeColumnIDs={activeColumnIDs} className="pf-v5-c-table__action">
+          <DropdownWithKebab obj={obj} />
+        </TableData>
+      </>
+    );
+  };
+
+  return (
+    <VirtualizedTable<K8sResourceCommon>
+      data={data}
+      unfilteredData={unfilteredData}
+      loaded={loaded}
+      loadError={loadError}
+      columns={columns}
+      Row={AllPolicyRow}
+    />
+  );
+};
+
+type DropdownWithKebabProps = {
+  obj: K8sResourceCommon;
+};
+
+const DropdownWithKebab: React.FC<DropdownWithKebabProps> = ({ obj }) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const onSelect = (_event: React.MouseEvent<Element, MouseEvent> | undefined, value: string | number | undefined) => {
+    // eslint-disable-next-line no-console
+    console.log('selected', value);
```

Comment:
must see what other plugins are doing with logging. Some of the core console stuff seems to be using some sort of fancy/decorated logger, perhaps with levels. just a thought, requesting no action